support-frontend/assets/components/ticker/contributionTicker.tsx

```diff
@@ -171,17 +171,13 @@ export default class ContributionTicker extends Component<
 			);
 		}
 
-		if (this.state.goalReached && this.props.tickerEndType === 'hardstop') {
```

Comment:
We don't need this check at all we've already returned if `!goalReached` or `tickerEndType !== 'hardstop'`